### ISSUE TITLE
Error when follow tutorials [15 - EMR.ipynb] 

### ISSUE THREAD

**Issue by jiajie999** [awslabs/aws-data-wrangler#252]
**Describe the bug**
I have followed the tutorial [15 - EMR.ipynb](https://github.com/awslabs/aws-data-wrangler/blob/master/tutorials/15%20-%20EMR.ipynb) and get an error.
![f1](https://user-images.githubusercontent.com/1141716/82561095-5a975e00-9ba5-11ea-8ae4-4c508e1fb831.png)

After some digging, the reason is wrong region name 'ap-south-' in docker repo path.  
![f2](https://user-images.githubusercontent.com/1141716/82561107-608d3f00-9ba5-11ea-8037-03caaf8da114.png)

**To Reproduce**
Just follow instruction from  [15 - EMR.ipynb](https://github.com/awslabs/aws-data-wrangler/blob/master/tutorials/15%20-%20EMR.ipynb) in aws region ap-south-1, will get error.

**Comment by igorborgest** [comment]
Hi @jiajie999 !

Thanks for the fix. I also added some others improvements.

It's already in the `dev` branch, please could you test it?

```bash
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

The idea is to release it on version `1.2.1`.

**Comment by jiajie999** [comment]
> Hi @jiajie999 !
> 
> Thanks for the fix. I also added some others improvements.
> 
> It's already in the `dev` branch, please could you test it? (Everything is working here)
> 
> ```shell
> pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
> ```
> 
> The idea is to release it on version `1.2.1`.

Hi @igorborgest!

I have tested the dev branch, everything works fine to me.
happy to help :)

**Comment by igorborgest** [comment]
Changes released on version [1.3.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.3.0)

Thanks!
